trillian-logsigner.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: trillian-logsigner
+  version: 1.5.2
+  epoch: 0
+  description: Merkle tree implementation used in Sigstore
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - go
+
+pipeline:
+  - uses: go/install
+    with:
+      package: github.com/google/trillian/cmd/trillian_log_signer
+      version: v${{package.version}}
+
+update:
+  enabled: true
+  github:
+    identifier: google/trillian
```

Comment:
Does it make sense for one of these to be a subpackage of the other, so they can both be updated as one update when the repo has a new release?

I think we'd just pick one to promote and make the other a subpackage.